fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -677,6 +731,7 @@ private void removeEmptyFragments() {
                         throw new RepositoryRuntimeException("Error removing empty fragments", ex);
                     }
                 });
+                touch();
```

Comment:
I was thinking the same thing, but thought it may be useful to leave this `touch` in place to mitigate new calls to `removeEmptyFragments()`... but I am also fine to see it removed.